Products/CMFPlone/patterns/tinymce.py

```diff
@@ -94,6 +94,12 @@ def get_all_style_formats(self):
 
     def get_tiny_config(self):
         settings = self.settings
+
+        # NOTE: The `importcss_file_filter` is used to filter the CSS files
+        # from `content_css` which should be used to automatically create the
+        # styles dropdown.
+        # Also see:
+        # https://www.tiny.cloud/docs/tinymce/latest/importcss/#importcss_file_filter
```

Comment:
```suggestion
        # https://6.docs.plone.org/classic-ui/tinymce-customization.html#inject-formats-with-files-named-tinymce-formats-css
```

I'd put the plone docs url here ...